graf3d/eve7/inc/ROOT/REveCaloData.hxx

```diff
@@ -204,11 +204,11 @@ public:
    virtual Float_t GetMaxVal(Bool_t et) const { return et ? fMaxValEt : fMaxValE; }
    Bool_t  Empty() const { return fMaxValEt < 1e-5; }
 
-   virtual TAxis*  GetEtaBins()    const { return fEtaAxis; }
-   virtual void    SetEtaBins(TAxis* ax) { fEtaAxis=ax; }
+   virtual TAxis*  GetEtaBins()    const { return fEtaAxis.get(); }
+   virtual void    SetEtaBins(TAxis* ax) { fEtaAxis.reset(ax); }
```

Comment:
Better use `unique_ptr` like:
```
   virtual void    SetEtaBins(std::unique_ptr<TAxis> &ax) { fEtaAxis = ax; }

```
Then it will be clear that caller have to provide object with ownership